heron/config/src/yaml/conf/local/healthmgr.yaml

```diff
@@ -0,0 +1,10 @@
+## list of policies to be executed for self regulation
+#heron.class.health.policies:
+#  - dynamic-resource-allocation
+#
+## configuration specific to individual policies listed above
+#dynamic-resource-allocation:
+#  health.policy.class: "com.twitter.heron.healthmgr.policy.DynamicResourceAllocationPolicy"
+#  health.policy.interval.ms: "120000"
```

Comment:
all other heron yaml configs in this directory and elsewhere leverage primitive ints so we should provide a consistent user experience. Actually we don't use quotes around strings or numbers elsewhere, so I think we should remove them here to not be the exception. Also asking users to use quotes where not necessary is cumbersome.